### PR TITLE
Add icons to template-chooser

### PR DESCRIPTION
**Pull request by Abdallah-Moh**
Add required icons to the `template-chooser` branch

### CODE REVIEW

**Review by Suzie97 — CHANGES_REQUESTED** [review]
Some lines for the facebook icons have been repeated in the gresource file. The rest of the code looks good to me.

File: data/gresource.xml [hunk 1]
```diff
@@ -2,5 +2,77 @@
...
+    <file alias="instagram_profile photo.svg">icons/hicolor/instagram_profile photo.svg</file>
+    <file alias="instagram_square.svg">icons/hicolor/instagram_square.svg</file>
+    <file alias="instagram_stories.svg">icons/hicolor/instagram_stories.svg</file>
+    <file alias="facebook_cover photo.svg">icons/hicolor/facebook_cover photo.svg</file>
+    <file alias="facebook_landscape.svg">icons/hicolor/facebook_landscape.svg</file>
+    <file alias="facebook_portrait.svg">icons/hicolor/facebook_portrait.svg</file>
+    <file alias="facebook_profile photo.svg">icons/hicolor/facebook_profile photo.svg</file>
+    <file alias="facebook_square.svg">icons/hicolor/facebook_square.svg</file>
+    <file alias="facebook_stories.svg">icons/hicolor/facebook_stories.svg</file>
+    <file alias="facebook_cover_photo.svg">icons/hicolor/facebook_cover_photo.svg</file>
```
[the remark this answers is not in the document]
Comment (Abdallah-Moh) [reply data/gresource.xml]: My bad ran the command twice

File: data/gresource.xml [hunk 2]
```diff
@@ -2,5 +2,68 @@
...
     <file alias="styles.css">styles.css</file>
+    <file alias="paper-a0.svg">icons/hicolor/paper_a0.svg</file>
+    <file alias="paper-a1.svg">icons/hicolor/paper_a1.svg</file>
+    <file alias="paper-a2.svg">icons/hicolor/paper_a2.svg</file>
+    <file alias="paper-a3.svg">icons/hicolor/paper_a3.svg</file>
+    <file alias="paper-a4.svg">icons/hicolor/paper_a4.svg</file>
+    <file alias="paper-a5.svg">icons/hicolor/paper_a5.svg</file>
+    <file alias="paper-a6.svg">icons/hicolor/paper_a6.svg</file>
+    <file alias="paper-a7.svg">icons/hicolor/paper_a7.svg</file>
+    <file alias="photo_2x3.svg">icons/hicolor/photo_2x3.svg</file>
```
Comment (Suzie97) [inline data/gresource.xml:13]: Can we use hyphens (-) instead of underscore( _ ) for these names?

File: data/gresource.xml [hunk 3]
```diff
@@ -2,5 +2,68 @@
...
+    <file alias="photo_4x6.svg">icons/hicolor/photo_4x6.svg</file>
+    <file alias="photo_5x7.svg">icons/hicolor/photo_5x7.svg</file>
+    <file alias="photo_8.5x11.svg">icons/hicolor/photo_8.5x11.svg</file>
+    <file alias="photo_8x10.svg">icons/hicolor/photo_8x10.svg</file>
+    <file alias="photo_12x18.svg">icons/hicolor/photo_12x18.svg</file>
+    <file alias="photo_24x36.svg">icons/hicolor/photo_24x36.svg</file>
+    <file alias="instagram_landscape.svg">icons/hicolor/instagram_landscape.svg</file>
+    <file alias="instagram_portrait.svg">icons/hicolor/instagram_portrait.svg</file>
+    <file alias="instagram_profile photo.svg">icons/hicolor/instagram_profile photo.svg</file>
```
Comment (Suzie97) [inline data/gresource.xml:22]: There is a space between profile and photo here.

**Review by Suzie97 — COMMENTED** [review]
@Abdallah-Moh, can we use hyphen (-) instead of underscore ( _ ) for the icon names?

File: data/gresource.xml (hunk 2 above)
Comment (Abdallah-Moh) [reply data/gresource.xml:13]: ok

**Pull request merged**